### PR TITLE
i3exit: add page

### PR DESCRIPTION
**Pull request by marchersimon**

### CODE REVIEW

File: pages/linux/i3exit.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Log out of i3:
+
+`i3exit logout`
+
+- Lock i3:
+
+`i3exit lock`
+
+- Shut down the machine:
```
Comment (navarroaxel) [inline pages/linux/i3exit.md:14]: We use `system` in `poweroff` and `reboot` pages
```suggestion
- Shut down the system:
```

File: pages/linux/i3exit.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`i3exit shutdown`
+
+- Suspend the machine:
+
+`i3exit suspend`
+
+- Switch the current user:
```
Comment (navarroaxel) [inline pages/linux/i3exit.md:22]: Switch to what? :thinking:

Comment (marchersimon) [reply pages/linux/i3exit.md:22]: I logs you out and prompts you to choose a user. It's the same as in every DE when you click on "switch user".

Comment (navarroaxel) [reply pages/linux/i3exit.md:22]: It's not the same goes to the select user screens keeping the current session open, than log off and go to the select user to login screen. It's nice to leave that clear here.

File: pages/linux/i3exit.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Lock i3:
+
+`i3exit lock`
+
+- Shut down the system:
+
+`i3exit shutdown`
+
+- Suspend the machine:
```
Comment (navarroaxel) [inline pages/linux/i3exit.md:18]:
```suggestion
- Suspend the system:
```

File: pages/linux/i3exit.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+`i3exit suspend`
+
+- Switch the current user:
+
+`i3exit switch_user`
+
+- Hibernate the machine:
```
Comment (navarroaxel) [inline pages/linux/i3exit.md:26]:
```suggestion
- Hibernate the system:
```

File: pages/linux/i3exit.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+`i3exit hibernate`
+
+- Reboot the machine:
```
Comment (navarroaxel) [inline pages/linux/i3exit.md:30]:
```suggestion
- Reboot the system:
```

File: pages/linux/i3exit.md (hunk 2 above)
Comment (marchersimon) [reply pages/linux/i3exit.md:22]: Well, I'm not actually sure. I tried it, but it just showed me a blank screen and crashed. But what it does is calling `dm-tool switch-to-greeter`. From the man pages

> Switch to the greeter suitable for logging into a new session.

Sounds like it doesn't log you out. How about we change the description to
```suggestion
- Switch to the login screen to log in as a different user:
```

Comment (sbrl) [reply pages/linux/i3exit.md:22]: It's sounds like this is essentially a "switch user" feature. @marchersimon's suggestion seems logical to me.

**Review by navarroaxel — APPROVED** [review]

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**